lib/rubocop/cop/rspec/not_to_not.rb

```diff
@@ -24,14 +25,12 @@ class NotToNot < Cop
 
         def on_send(node)
           not_to_not_offense(node, alternative_style) do
-            add_offense(node, location: :selector)
+            add_offense(node.loc.selector) do |corrector|
```

Comment:
The `add_offense` should have the `message` parameter:
```suggestion
            add_offense(node.loc.selector, message: format(MSG, replacement: style, original: alternative_style)) do |corrector|
```

Reply:
If no `message` parameter is passed, RuboCop will call the `#message` method which returns a string. I prefer the more explicit version, but the callback approach still works.

Reply:
Ok, got it.
I was just checking the rest of the Changelog, and almost everywhere the code is now calling `add_offense` with the `message` parameter, except when there is no interpolation, and the `MSG` constant is being used automatically.

Maybe this is something, that can be aligned afterwards as well.
So that the code is in line with the expectation from [`add_offense`](https://github.com/rubocop-hq/rubocop/blob/v0.87.1/lib/rubocop/cop/internal_affairs/redundant_message_argument.rb#L12-L21)